crates/napi/src/bindgen_runtime/js_values/number.rs

```diff
@@ -49,16 +50,20 @@ macro_rules! impl_number_conversions {
             $name,
           )?;
 
-            Ok(ret)
-				}
+          Ok(ret.try_into().expect(concat!("Failed to convert ", stringify!($st), " to ", stringify!($t))))
```

Comment:
Since this method already returns a `Result`, should this return an `Err` instead of panicking when conversion failed?

Reply:
Done.